.ci/docker/common/install_rocm.sh

```diff
@@ -91,7 +91,7 @@ install_centos() {
   echo "gpgcheck=1" >> /etc/yum.repos.d/amdgpu.repo
   echo "gpgkey=http://repo.radeon.com/rocm/rocm.gpg.key" >> /etc/yum.repos.d/amdgpu.repo
 
-  local rocm_baseurl="http://repo.radeon.com/rocm/yum/${ROCM_VERSION}"
+  local rocm_baseurl="http://repo.radeon.com/rocm/yum/${ROCM_VERSION}/main"
```

Comment:
@oraluben Can you provide some pointers to why this change is needed?

Reply:
@jithunnair-amd 
The url changed since ROCm 5.1
Before 5.1: http://repo.radeon.com/rocm/yum/5.0.2/
After: http://repo.radeon.com/rocm/yum/5.1/main/

You can check it by visiting urls above.

Reply:
That might work up until ROCm 6.3.  The URL seems empty?  https://repo.radeon.com/rocm/yum/6.3/main/  

ROCm 6.2.4 packages are under main.  https://repo.radeon.com/rocm/yum/6.2.4/main/

RHEL8 and RHEL9 have current ROCm support.

Reply:
> That might work up until ROCm 6.3. The URL seems empty?

This sounds like an AMD issue, could you confirm that internally?